jumanji/environments/routing/cleaner/generator.py

```diff
@@ -62,12 +65,30 @@ def __call__(self, key: chex.PRNGKey) -> Maze:
             key: the Jax random number generation key.
 
         Returns:
-            maze: the generated maze.
+            state: the generated state.
         """
-        maze = maze_generation.generate_maze(self.num_rows, self.num_cols, key)
-        return self._adapt_values(maze)
+        generator_key, state_key = jax.random.split(key)
+        maze = maze_generation.generate_maze(
+            self.num_rows, self.num_cols, generator_key
+        )
+
+        grid = self._adapt_values(maze)
+
+        # Agents start in upper left corner
+        agents_locations = jnp.zeros((self.num_agents, 2), int)
+
+        # Clean the tile in upper left corner
+        grid = grid.at[0, 0].set(CLEAN)
+
+        return State(
+            grid=grid,
+            agents_locations=agents_locations,
+            action_mask=None,
+            step_count=jnp.array(0, jnp.int32),
+            key=state_key,
+        )
 
-    def _adapt_values(self, maze: Maze) -> Maze:
+    def _adapt_values(self, maze: maze_generation.Maze) -> chex.Array:
```

Comment:
Isn't the return still a `Maze`?

Reply:
It returns what I called a Maze before (which was an alias for chex.Array), however in the state it is typed as a chex.Array, do you think I should create aliases in the state ? It would also mean alias for the action mask and the agent locations to be consistent...

Reply:
In that case I would keep the return type as is. Do you think we should keep the type annotation of `maze_generation.Maze` for `maze` or just `chex.Array`?